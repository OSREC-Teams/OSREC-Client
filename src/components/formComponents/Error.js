import styled from 'styled-components'

export default styled.div`
  color: white;
  background: red;
  height: 2em;
  padding: 0.5em;
  font-family: 'Exo 2', sans-serif;
  width: 100%;
  border-radius: 5px;
  margin-bottom: 10px;
  width-min: 40px;
  display: flex;
  align-items: center;
  justify-content: evenly;
  flex-direction: column;
`;
