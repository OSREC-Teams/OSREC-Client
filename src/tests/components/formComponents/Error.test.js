import React from 'react';
import Error from 'components/formComponents/Error';
import { shallow } from 'enzyme';

test('Button renders a form error div', () => {
  const error = shallow(<Error>Error</Error>);
  expect(error).toMatchSnapshot();
});
