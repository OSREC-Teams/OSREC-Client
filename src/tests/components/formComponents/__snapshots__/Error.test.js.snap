// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders a form error div 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  color: white;
  background: red;
  height: 2em;
  padding: 0.5em;
  font-family: 'Exo 2', sans-serif;
  width: 100%;
  border-radius: 5px;
  margin-bottom: 10px;
  width-min: 40px;
  display: flex;
  align-items: center;
  justify-content: evenly;
  flex-direction: column;
",
        ],
      },
      "displayName": "styled.div",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Error
</StyledComponent>
`;
